app/models/grda_warehouse/hud/client.rb

```diff
@@ -743,6 +743,12 @@ def self.exists_with_inner_clients(inner_scope)
           and(
             arel_table[:consent_form_signed_on].gteq(consent_validity_period.ago)
           ))
+      when 'Two Years'
+        where(
+          arel_table[:housing_release_status].matches("%#{full_release_string}").
+          and(
+            arel_table[:consent_form_signed_on].gteq(consent_validity_period.ago)
+          ))
```

Comment:
Since this uses `consent_validity_period`, maybe combine with the previous alternative?
`when 'One Year', 'Two Years'`?